login-render-engine-impl/pack/src/webapp/vm/defaultskin/xlogin.vm

```diff
@@ -30,6 +30,10 @@
 				</td>
 			</tr>
 			<tr>
+				<td colspan="2"><p class="instruction" align="center">${ssoWording}&nbsp;<a href="${logInOutUrl}">${ssoUrlWording}</a></p>
```

Comment:
So that we can push this change back to Sakai you should have a test to see if these values are present in the context and only output the HTML if they are.